packages/react/src/components/ExpressiveModal/ExpressiveModal.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import { ComposedModal } from 'carbon-components-react';
+import ExpressiveModalCloseBtn from './ExpressiveModalCloseBtn';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Expressive Modal
+ *
+ * @param {object} props react proptypes
+ * @returns {object} JSX object
+ */
+const ExpressiveModal = ({ open, className, children, onClose, ...props }) => {
+  const [isOpen, setIsOpen] = useState(open);
+  const [prevOpen, setPrevOpen] = useState(open);
+
+  useEffect(manageOpenState);
+
+  return (
+    <ComposedModal
+      onClose={onClose}
+      open={isOpen}
+      data-autoid={`${stablePrefix}--expressive-modal`}
+      className={classNames(`${prefix}--modal--expressive`, className)}
+      {...props}>
+      <ExpressiveModalCloseBtn onClick={closeModal} />
+      {children}
+    </ComposedModal>
+  );
+
+  /**
+   * Close modal
+   */
+  function closeModal() {
+    if (!onClose || onClose() !== false) {
+      setIsOpen(false);
+    }
+  }
+
+  /**
+   * Manage open prop and isOpen state
+   */
+  function manageOpenState() {
+    if (prevOpen !== open) {
+      setIsOpen(open);
+      setPrevOpen(open);
+    }
+  }
+};
+
+ExpressiveModal.propTypes = {
+  open: PropTypes.bool,
+  className: PropTypes.string,
+  children: PropTypes.node,
+  onClose: PropTypes.func,
+};
+
+ExpressiveModal.defaultProps = {
+  open: false,
```

Comment:
@photodow I'm wondering if we should also pass in props for the primary and secondary buttons similar to how the productive modal works. They can be optional attributes, and if they don't pass anything then the ButtonGroup doesn't render.

cc: @annawen1 @sannminwin @andysherman2121 @kennylam

Reply:
No, I don't think so. The modal itself isn't supposed to have buttons. When you add buttons it becomes a dialogue. The locale selector and lightbox, for example, should use this component, but won't have buttons.

@wonilsuhibm @oliviaflory please correct me if I misunderstood.

Reply:
@jeffchew Wonil and I met with James on Friday and decided the expressive modal should just be a shell so adopters can add whatever they need to it. We will still provide the other modals Carbon has in the theming. Sorry for not updating our convo in the original issue.

After a design critique with Carbon on Thursday we think the expressive modal is different than the current modals that exist in Carbon today, which are like a 'dialogue box' that requires an action / decision. Whereas the expressive modal could be used for lightbox or local selector that could have different requirements or no action required. (We need to work on the definition).

Reply:
Thanks for bringing me up to speed @oliviaflory . It's too bad we can't just call ours `<Modal />` and rename theirs `<Dialog />`